src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,8 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
-  {{#if app-launcher-menu-item-external-icon--attribute}}
-	  {{{app-launcher-menu-item-external-icon--attribute}}}
-  {{/if}}></i>
+<span class="pf-c-app-launcher__menu-item-external-icon">
+  <i class="fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
```

Comment:
Now that the `<span>` is the BEM element, can you move the `{{#if}}` logic for both `app-launcher-menu-item-external-icon--modifier` and `app-launcher-menu-item-external-icon--attribute` to the span?

Reply:
I was going to break it up into separate lines for readability like this
![Screen Shot 2020-03-30 at 16 26 00](https://user-images.githubusercontent.com/35377991/77958626-4e78d980-72a3-11ea-8199-dcf2882333cf.png)

Reply:
And fixed the next item (for line 319)